packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.test.ts

```diff
@@ -308,7 +308,7 @@ describe("DefaultVisualizers unit tests", () => {
 		sharedMatrix.setCell(0, 0, "Hello");
 		sharedMatrix.setCell(0, 1, "World");
 		// False positive
-		// eslint-disable-next-line unicorn/no-useless-undefined
+
```

Comment:
If the linter doesn't complain about this anymore for some reason, let's remove the "False positive" comment right above.

Reply:
Good catch.